rust/src/builtins/scriptlet_intercept/usermod.rs

```diff
@@ -34,9 +34,8 @@ pub(crate) fn entrypoint(args: &[&str]) -> Result<()> {
     };
 
     let rootdir = Dir::open_ambient_dir("/", cap_std::ambient_authority())?;
-    let conf_dir = common::open_create_sysusers_dir(&rootdir)?;
     for secondary_group in groups {
-        generate_sysusers_fragment(&conf_dir, username, secondary_group)?;
```

Comment:
I'm probably missing something obvious here, but I don't follow how this changes a behaviour. Aren't we calling `common::open_create_sysusers_dir()` on the same root dir either way?